warehouse/query-core/src/test/java/datawave/query/language/parser/jexl/TestLuceneToJexlParser.java

```diff
@@ -9,71 +8,62 @@
 public class TestLuceneToJexlParser {
     
     @Test
-    public void test1() {
```

Comment:
This is beyond the scope of your changes and won't prevent an approval, but it would be nice to see well named tests (perhaps with comments).